services/src/main/java/com/linkedin/kmf/services/configs/ProduceServiceConfig.java

```diff
@@ -92,7 +120,32 @@
                                     ConfigDef.Type.INT,
                                     5,
                                     ConfigDef.Importance.LOW,
-                                    PRODUCE_THREAD_NUM_DOC);
+                                    PRODUCE_THREAD_NUM_DOC)
+                            .define(AUTO_TOPIC_MIN_ISR_CONFIG,
+                                    ConfigDef.Type.INT,
+                                    -1,
```

Comment:
OK.  I will remove the config and set min.isr to max(replicationFactor -1, 1) when creating the auto topic.